main.py

```diff
@@ -1,15 +1,20 @@
 from sketchy import Sketchy
 from ball import Ball
+from vector import Vector
 
 class MyDrawing(Sketchy):
 
     def setup(self):
-        return
+        width, height = [600, 400]
```

Comment:
I am more so using ball.py and main.py to play around and see what i can do using the vector library.